File: src/c#/Elevator/PrimitiveElevatorTest.cs
using System;
using Elevator.domain;
namespace Elevator {
    public class PrimitiveElevatorTest {
		public static void Main(String[] args)
		{
			PrimitiveElevator primitiveElevator = new PrimitiveElevator();

			primitiveElevator.OpenDoor();
			primitiveElevator.CloseDoor();
			primitiveElevator.GoDown();
			primitiveElevator.GoUp();
			primitiveElevator.GoUp();
			primitiveElevator.OpenDoor();
			primitiveElevator.CloseDoor();
			primitiveElevator.GoDown();
			primitiveElevator.OpenDoor();
			primitiveElevator.GoDown();
			primitiveElevator.CloseDoor();
			primitiveElevator.GoDown();
			primitiveElevator.GoDown();

			int curFloor = primitiveElevator.GetFloor();

			if (curFloor != 5 && !primitiveElevator.GetDoorStatus())
			{
				primitiveElevator.Go(5);
			}

			primitiveElevator.Go(-10000000);
			primitiveElevator.OpenDoor();
		}
	}
}

File: src/java/org/javaturk/dp/ch02/elevator/PrimitiveElevatorTest.java
package org.javaturk.dp.ch02.elevator;

import org.javaturk.dp.ch02.elevator.domain.PrimitiveElevator;

public class PrimitiveElevatorTest {

	public static void main(String args[]) {

		PrimitiveElevator primitiveElevator = new PrimitiveElevator();

		primitiveElevator.openDoor();
		primitiveElevator.closeDoor();
		primitiveElevator.goDown();
		primitiveElevator.goUp();
		primitiveElevator.goUp();
		primitiveElevator.openDoor();
		primitiveElevator.closeDoor();
		primitiveElevator.goDown();
		primitiveElevator.openDoor();
		primitiveElevator.goDown();
		primitiveElevator.closeDoor();
		primitiveElevator.goDown();
		primitiveElevator.goDown();

		int curFloor = primitiveElevator.getFloor();

		if (curFloor != 5 && !primitiveElevator.getDoorStatus()) {
			primitiveElevator.go(5);
		}

		primitiveElevator.go(-1000_0000);
		primitiveElevator.openDoor();
	}
}
